DEVELOPERS.md

```diff
@@ -235,12 +215,37 @@ panel: "human-sc-immunology-spatial-proteomics"
 panel_file: null
 ```
 
-## Pixelator benchmark tests
 
+#### Pixelator nf-core/pixelator integration tests
+
+Pixelator is build to be orchestrated by the nextflow pipeline nf-core/pixelator.
+This means that is is useful to have a simple way to test the integration between the two.
+
+You can do this using tasks:
+
+```shell
+task test-nf-core-pixelator
+```
+
+It can also be triggered on Github Actions:
+
+```
+gh workflow run --ref <your-branch-name> nf-core-pixelator-tests.yml
+```
+
+
+### Pixelator benchmark tests
+
+Sometimes it is useful to be able to run micro-benchmarks to see how changes affect performance.
 Pixelator uses `pytest-benchmark` to enable running micro-benchmarks. Normally, when running the tests these are disabled.
-You can enable them by running `pytest --benchmark-enable tests/`.
 
-## Pytest markers
+To run these test use:
+
+```shell
+task test-benchmark
+```
+
+### Pytest markers
 
 Pixelator defines several additional markers for tests.
 These are usually disabled by default and can be enabled by running `pytest -m <marker> tests/`
```

Comment:
This is the first time the pytest command is mentioned. For a Python noob like me It'd be helpful to understand that commands like `task test` and `task test-all` actually runs pytest, and that these commands can be found in the Taskfile.yml.

Reply:
I will look at this. To me the idea is that we should be able to abstract away a lot of the details of e.g. how things are run using the task file, so that for the most part the user developer doesn't have to worry about that.

Reply:
Maybe it's best to just keep it as it is. Just wanted to highlight something that caught my eye ;-)

Reply:
I have tried to make it a bit clearer in this version in the introduction to the testing section.